pkg/chunked/cache_linux.go

```diff
@@ -34,6 +34,8 @@ const (
 	// https://pages.cs.wisc.edu/~cao/papers/summary-cache/node8.html
 	bloomFilterScale  = 10 // how much bigger is the bloom filter than the number of entries
 	bloomFilterHashes = 3  // number of hash functions for the bloom filter
+
+	maxTagsLen = 100 * 1024 * 1024 // 100MB max size for tags len
```

Comment:
Why not build a Const for this magic number since the two PRs both create it.

Reply:
I've changed it to use the go-units package.